test/kube_testing/k8s.go

```diff
@@ -386,6 +413,21 @@ func (l *K8s) CreatePod(template *v1.Pod) *v1.Pod {
 	return results[0]
 }
 
+func (l *K8s) DeletePodsSafe(timeout int, pods ...*v1.Pod) {
```

Comment:
Please remove this function. Instead, change `DeletePods` to always pass a const timeout, defined appropriately in `const.go`.

Reply:
I am using podDeleteTimeout const which was already exists in k8s.go
Now force delete will be started in 15 seconds timeout after soft delete attempt for all pods